src/test/java/org/fundacionjala/pivotal/hooks/Hooks.java

```diff
@@ -13,8 +15,27 @@
     /**
      * After hook to delete the project created in tests.
      */
-    @After("@PostConditionProject")
-    public void afterCreatingProject() {
-        APIRequestManager.delete(APICommons.getEndPoint());
+    @After("@DeleteProject")
+    public void deleteProject() {
+        //System.out.println(APICommons.buildEndPoint("/projects/[Project.id]"));
+        APIRequestManager.delete(APICommons.buildEndPoint("/projects/[Project.id]"));
+    }
+
+    /**
+     * After hook to delete the project created in tests.
+     */
+    @After("@DeleteStory")
+    public void deleteStory() {
+        APIRequestManager.delete(APICommons.buildEndPoint("/projects/[Project.id]/stories/[Story.id]"));
+    }
+
+
+    /**
+     * After hook to delete the project created in tests.
+     */
+    @After("@deleteWorkspace")
+    public void deleteWorkspace() {
+        //System.out.println(APICommons.buildEndPoint("/projects/[Project.id]"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)